pixleesdk/src/main/java/com/pixlee/pixleesdk/client/PXLClient.kt

```diff
@@ -46,16 +46,16 @@ class PXLClient(val context: Context) {
         @Synchronized
         fun getInstance(context: Context): PXLClient {
             if (mInstance == null) {
-                mInstance = PXLClient(context)
+                mInstance = PXLClient()
+                android_id = Secure.getString(context.applicationContext.contentResolver, Secure.ANDROID_ID)
```

Comment:
So, instead, context is only used here in order to avoid PXLClient holding context.